### DISCUSSION TITLE
"Unsupported Media Type" on auth-with-password request

### DISCUSSION THREAD

**Discussion by coolroman** [pocketbase/pocketbase#2369] [Q&A] [answered]
I`m trying to authenticate by sending api request 

```
POST http://localhost:8090/api/collections/users/auth-with-password

{
  "identity": <email>.
  "password": <password>
}
```

and getting error:
```json
{
  "code": 400,
  "message": "An error occurred while loading the submitted data.",
  "data": {}
}
```

Error in logs:

|Name|Value|
|-- | --|
|Status | 400|
|Method | POST|
|Auth | guest|
|URL | /api/collections/users/auth-with-password|
|Referer | N/A|
|Remote IP | 127.0.0.1|
|User IP | 127.0.0.1|
|UserAgent | vscode-restclient|
|Meta | {   "errorDetails": "code=415, message=Unsupported Media Type",   "errorMessage": "An error occurred while loading the submitted data." }|
|Created | 2023-04-2711:55:28 UTC|

Pocketbase version: v0.15.2

**Comment by ganigeorgiev** [comment, marked as answer]
I'm not able to reproduce the issue. If you are not using the SDKs, make sure that you are sending a properly formatted JSON and have a `content-type:application/json` header sent with the request.

**Reply by coolroman** [reply]
Absence of `content-type:application/json` was the reason.
